tests/test_fips.py

```diff
@@ -98,6 +98,15 @@
     )
 
 
+def digest_xoflen(digest: str) -> str:
```

Comment:
Please add an explanation or a link to docs for those that are unfamiliar with cryptographic terms